sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSession.java

```diff
@@ -97,6 +98,8 @@
     private final AtomicReference<TransactionCoordinator> transactionCoordinator = new AtomicReference<>();
     private final Flux<AmqpShutdownSignal> shutdownSignals;
 
+    public static final Symbol CLIENT_ID = Symbol.getSymbol(AmqpConstants.VENDOR + ":receiver-name");
```

Comment:
Do we add this to sender client?

Reply:
Sorry, I forgot remove this in sender client.